pre-commit

```diff
@@ -68,7 +73,7 @@ for changedfile in $files_to_check; do
                 failures=$((failures + 1))
             fi
         elif [[ $(echo "$changedfile" | grep -q '\.*\.pp$'; echo $?) -eq 0 ]]; then
-            ${subhook_root}/puppet_manifest_syntax_check.sh "$changedfile"
+            ${subhook_root}/puppet_manifest_syntax_check.sh "$changedfile" "" "$USE_PUPPET_FUTURE_PARSER"
```

Comment:
$2 is module dir (https://github.com/drwahl/puppet-git-hooks/blob/master/commit_hooks/puppet_manifest_syntax_check.sh)
but its not used with the pre-commit hook; only with the pre-receive hook